quipucords/cli/auth/tests.py

```diff
@@ -24,16 +35,96 @@ def setUp(self):
         # nosetests command.
         self.orig_stderr = sys.stderr
         sys.stderr = HushUpStderr()
+        if os.path.isfile(TMP_KEY):
+            os.remove(TMP_KEY)
+        with open(TMP_KEY, 'w') as test_sshkey:
+            test_sshkey.write('fake ssh keyfile.')
 
     def tearDown(self):
         # Restore stderr
         sys.stderr = self.orig_stderr
+        if os.path.isfile(TMP_KEY):
+            os.remove(TMP_KEY)
 
-    def test_add_auth(self):
-        """Testing the add auth command"""
-        auth_out = six.StringIO()
-        with redirect_stdout(auth_out):
+    def test_add_req_args_err(self):
+        """Testing the add auth command required flags"""
+        with self.assertRaises(SystemExit):
             sys.argv = ['/bin/qpc', 'auth', 'add', '--name', 'auth1']
             CLI().main()
-            self.assertEqual(auth_out.getvalue(),
-                             'Auth "auth1" was added\n')
+
+    def test_add_bad_key(self):
+        """Testing the add auth command when providing an invalid path for
+        the sshkeyfile.
+        """
+        auth_out = six.StringIO()
```

Comment:
Question: since quipucords the server is requiring Python 3.5 or greater, does the CLI require Python 3 as well?

If yes, no need to use `six` here.